engines/wintermute/base/gfx/opengl/base_surface_opengl3d.cpp

```diff
@@ -128,8 +128,27 @@ bool BaseSurfaceOpenGL3D::create(const Common::String &filename, bool defaultCK,
 	}
 
 	if (needsColorKey) {
-		Graphics::TransparentSurface trans(*_surf);
-		trans.applyColorKey(_ckRed, _ckGreen, _ckBlue, replaceAlpha);
+		// this is taken from Graphics::TransparentSurface
+		// only difference is that we set the pixel
+		// color to transparent black, like D3DX,
+		// if it matches the color key
+		for (int i = 0; i < _surf->h; i++) {
```

Comment:
May I suggest using x and y as iteration variables (as appropriate), instead of i and j. More meaningful, and easier to verify that they make sense then

Reply:
Done